configs/repos/common/packages/base-env/package.py

```diff
@@ -21,19 +20,19 @@ class BaseEnv(BundlePackage):
 
     # Basic utilities
     if sys.platform == 'darwin':
-        depends_on('libbacktrace', type=('build', 'run'))
-    depends_on('cmake', type=('build', 'run'))
+        depends_on('libbacktrace', type='run')
+    depends_on('cmake', type='run')
     depends_on('git', type='run')
 
     # I/O
-    depends_on('zlib', type=('build', 'run'))
-    depends_on('szip', type=('build', 'run'))
-    depends_on('hdf5', type=('build', 'run'))
-    depends_on('netcdf-c', type=('build', 'run'))
-    depends_on('netcdf-fortran', type=('build', 'run'))
-    depends_on('parallel-netcdf', type=('build', 'run'))
-    depends_on('parallelio', type=('build', 'run'))
-    depends_on('nccmp', type=('build', 'run'))
+    depends_on('zlib', type='run')
+    depends_on('szip', type='run')
+    depends_on('hdf5', type='run')
+    depends_on('netcdf-c', type='run')
+    depends_on('netcdf-fortran', type='run')
+    depends_on('parallel-netcdf', type='run')
+    depends_on('parallelio', type='run')
+    depends_on('nccmp', type='run')
```

Comment:
Are these not build dependencies?

Reply:
The thing is, none of these "envs" builds anything. The build dependencies are in each of the packages that get installed. For the envs, you only need the "run" dependency so that the modules get loaded when the env module gets loaded.